04/dbernhard/tests_if6.oat

```diff
@@ -0,0 +1,15 @@
+int program(int argc, string[] args) {
+    return 2*function(0) + function(1) + function(2);
+}
+
+int function(int x) {
+    if (x == 0) {
+        return 1;
+    } else if (x == 1) {
+        return 0;
+    } else if (x == 2) {
+        return 0;
+    } else {
+        return 0;
```

Comment:
Same comment, for each different branch I would return a different value in a way that we can verify that each function call has taken a different branch from the result